src/config/enterprise.rs

```diff
@@ -237,7 +237,7 @@ impl<'a> PipelinesVersionPayload<'a> {
         Self {
             data: PipelinesData {
                 attributes: PipelinesAttributes {
-                    config_hash: fields.config_version,
+                    config_hash: fields.config_version_hash,
```

Comment:
Same here. We are sending a `config_version_hash` to a `config_hash` endpoint.

I realise this is a necessity right now given that we're still reporting to McNulty (and we definitely don't want to waste time updating McNulty), but for now `config_version_hash` should be `configuration_version_hash` and `config_hash` (the actual key/value that makes up the payload) can be updated in the API per https://datadoghq.atlassian.net/browse/OP-400